packages/components/modules/profiles/web/ProfileMembers/constants.ts

```diff
@@ -13,6 +13,10 @@ export enum MemberRoles {
   manager = 'MANAGER',
 }
 
+export enum MemberActions {
```

Comment:
we're trying to avoid enums because they have a lot of [bugs](https://github.com/microsoft/TypeScript/issues?q=is%3Aissue%20state%3Aopen%20enum). Having a const + types is often simpler and safer. If you have some time, could you convert `MemberStatuses` and `MemberRoles`? 🙏🏽 
```suggestion
export const MEMBER_ACTIONS {
  remove = 'REMOVE',
}

export type MemberActions = ValueOf<typeof  MEMBER_ACTIONS>
```